cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -453,6 +453,66 @@ describe("Timeline", () => {
             cy.get(".mx_RoomView_searchResultsPanel").percySnapshotElement("Highlighted search results");
         });
 
+        it("should find words in various languages and highlight them on search result", () => {
+            // "Test" in various languages
+            const stringAr = "اِمْتِحَان‎";
+            const stringEl = "εξέταση";
+            const stringEn = "Test";
+            const stringHe = "מִבְחָן";
+            const stringHi = "आज़माइश";
+            const stringJa = "テスト";
+            const stringKo = "테스트";
+            const stringTh = "การสอบ";
+            const stringZh = "测试";
+
+            const sendString = (string: string) => {
+                cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                    msgtype: "m.text" as MsgType,
+                    body: string,
+                });
+            };
+
+            const searchAndHighlight = (string: string) => {
+                cy.get(".mx_SearchBar_input input").clear().invoke("val", string).trigger("input").type("{enter}");
+                cy.get(".mx_EventTile:not(.mx_EventTile_contextual) .mx_EventTile_searchHighlight").should("exist");
```

Comment:
It seems to me that, if this fails, it will be unclear which string caused it to fail. could we add a `cy.log` to say what we are searching for?